.github/workflows/pr-checks.yaml

```diff
@@ -14,7 +14,7 @@ env:
   K8S_VERSION: "1.19.2"
   MINIKUBE_VERSION: "1.15.1"
   TEST_ACCEPTANCE_CLI: "kubectl"
-  CONTAINER_RUNTIME: "docker"
+  CONTAINER_RUNTIME: "podman"
```

Comment:
not sure if we need to flip here the values because we do not use neither of them during the installation. Perhaps dropping the line completely?

Reply:
Good point, thx @pedjak